### ISSUE TITLE
CALL LENGTH exception

### ISSUE THREAD

**Issue by dust63** [CasparCG/Server#548]
Hi,

When I want to make this command

`CALL 1-1 LENGTH 250`

I received an exception:
```
[2017-02-02 14:33:38.256] [10864] [info]    Received message from Console: CALL 1-1 LENGTH 250\r\n
#502 CALL FAILED
[2017-02-02 14:33:38.260] [10300] [error]   producer\ffmpeg_producer.cpp(436): Throw in function class std::basic_string<wchar_t,struct std::char_traits<wchar_t>,class std::allocator<wchar_t> > __thiscall caspar::ffmpeg::ffmpeg_producer::do_call(const class std::basic_string<wchar_t,struct std::char_traits<wchar_t>,class std::allocator<wchar_t> > &)
Dynamic exception type: class boost::exception_detail::clone_impl<struct caspar::invalid_argument>
std::exception::what: Unknown exception
```

Regards.

**Comment by dimitry-ishenko** [comment]
@dust63 please provide more specifics

**Comment by dust63** [comment]
Hi Dimitry,

I send a command to play a file for example:

`PLAY 1-1 AMB LENGTH 400`

Caspar play the file correctly. After that I want to update the LENGTH of the current playing program on the layer 1.
So I send:

`CALL 1-1 LENGTH 250`

Caspar reply the exception mentionned above.

Can you try in your side to be sure it is a Caspar Server Error ?

**Comment by dust63** [comment]
I comfirm the Error was corrected on 2.1.0, the error is on the 2.0.7 version.

Sorry.

**Comment by dimitry-ishenko** [comment]
Ah OK
